containers/helm/values.yaml

```diff
@@ -0,0 +1,27 @@
+nameOverride: ""
+fullnameOverride: ""
+namespace: pwpush
+replicaCount: 1
+
+deployment:
+  labels:
+    app: "pwpush-ephemeral"
+    # component: ingress-controller
+    # type: nginx
+  containerPort: 5100
+  readinessProbe:
+    enabled: true
+    path: /
+  livenessProbe:
+    enabled: true
+    path: /
+
+image:
+  repository: docker.io/pglombardo/pwpush-ephemeral
+  pullPolicy: Always
+  tag: "latest"
```

Comment:
Let's point this to `release` since that's mostly stable.  `latest` breaks fairly often.